app/src/main/java/me/quadphase/qpdex/MoveSet.java

```diff
@@ -0,0 +1,52 @@
+package me.quadphase.qpdex;
+
+import java.util.List;
+
+//
+//
+//  Generated by StarUML(tm) Java Add-In
+//
+//  @ Project : QPdex (C) 2015
+//  @ File Name : MoveSet.java
+//  @ Date : 21-Jul-15
+//  @ Author : Nicole
+//
+//
+
+/**
+ * This is a wrapper class for the moves that a pokemon in your party has
+ */
+public class MoveSet {
+    /**
+     * Max: 4 per pokemon in the party
+     */
+    private List<Move> moves;
+
+    /**
+     * Constructor
+     */
+    public MoveSet(List<Move> moves) {
+        this.moves = moves;
+    }
+
+    /**
+     * adds a move to the moveset if it is not full
+     */
+    public boolean addMove(Move move) {
+        return true;
+    }
+
+    /**
+     * removes a move from the moveset
+     */
+    public boolean removeMove(Move move) {
```

Comment:
Might be easier to remove a Move by the index number if there's only a list of 4. But we can look into that as we begin to implement all the methods.